cmd/yanfd/main.go

```diff
@@ -167,16 +167,17 @@ func main() {
 			// Create multicast Ethernet face for interface
 			multicastEthTransport, err := face.MakeMulticastEthernetTransport(multicastEthURI, ndn.MakeDevFaceURI(iface.Name))
 			if err != nil {
-				core.LogFatal("Main", "Unable to create MulticastEthernetTransport for ", iface.Name, ": ", err)
-				os.Exit(2)
+				core.LogError("Main", "Unable to create MulticastEthernetTransport for ", iface.Name, ": ", err)
+				// os.Exit(2)
```

Comment:
Why not abort?

Reply:
On Windows some face creation will always fail. Also, I don't think it makes sense to exit the whole program only because one face does not work. The forwarder should be robust enough to work in the situation that part of the network interfaces are broken.

Reply:
Therefore,  I think the forwarder should run as a long as one face creation succeeds